tar_scm.py

```diff
@@ -885,6 +904,7 @@ def parse_args():
         args.submodules = True
     else:
         args.submodules = False
+    args.sslverify = True if args.sslverify == 'enable' else False
```

Comment:
The current code works fine, however please reverse the logic because it's more readable that way.